ui/src/pages/QuizPage.vue

```diff
@@ -0,0 +1,447 @@
+              <i class="bi bi-bookmark-heart" />
+            </button>
+          </div>
+
+          <div class="modal-body">
+            <div class="row">
+              <div class="col-4">
+                <div class="text-center">
+                  <img
+                    v-if="isCorrect"
+                    src="../assets/correct.png"
+                    class="answer-img"
+                    alt=""
+                  />
+                  <img
+                    v-else
+                    src="../assets/miss.png"
+                    class="answer-img"
+                    alt=""
+                  />
+                </div>
+              </div>
+
+              <div class="col-8">
+                <div v-if="isCorrect">
+                  <h3 class="answer-title">正解！</h3>
+                  <p>正解は{{ correctAnswer }}！</p>
+                  <p></p>
+                </div>
+
+                <div v-else>
+                  <h3 class="answer-title">残念...!</h3>
+                  <p>正解は{{ correctAnswer }}！</p>
+                  <p></p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="modal-footer">
+            <button 
+              v-show="currentQuizNumber < totalQuizNumber" 
+              type="button" class="btn btn-primary" 
+              data-dismiss="modal" 
+              @click="toNextQuiz()"
+            >
+              次の問題へ
+            </button>
+            <button 
+              v-if="currentQuizNumber == totalQuizNumber" 
+              type="button" class="btn btn-primary" 
+              data-toggle="modal" data-target="#answerEndedModal" 
+              @click="toNextQuiz()"
+            >
+              結果へ
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- すべての問題を回答し終わったときに表示するモーダル -->
+
+    <div
+      class="modal fade"
+      id="answerEndedModal"
+      tabindex="-1"
+      role="dialog"
+      aria-labelledby="exampleModalCenterTitle"
+      aria-hidden="true"
+      data-backdrop="static"
+    >
+      <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+          <div class="modal-header">おつかれさまでした！</div>
+
+          <div class="modal-body">
+            <div class="row">
+              <div class="col-4">
+                <div class="text-center">
+                  <img src="../assets/trophy.png" class="answer-img" />
+                </div>
+              </div>
+
+              <div class="col-8">
+                <p>正答数は{{ correctNumber.length }}/{{ totalQuizNumber }}</p>
+                <div v-for="(result, number) in results" :key="result">
+                  <div v-if="result">第{{ number+1 }}問 {{ quizzes[number]["CountryName"] }} 〇
+                    <button class="btn btn-light btn-circle" @click="updateBookmark(currentQuizNumber - 1)">
```

Comment:
>・結果画面のデザインをどうにかしたい(具体的にはブックマークの枠が気になるがどうしたらいいかわからない)

ブックマーク部分の背景色が、グレーっぽいのが気になるってことかな？
だとしたら、bootstrap の以下の指定を削除すれば解決するはず！
```suggestion
                    <button class="btn" @click="updateBookmark(currentQuizNumber - 1)">
```

Reply:
みんなが気にならないなら大丈夫かなとは思ったけど、普通はどんな感じがいいのかわかんないなと
思って！
一旦変えてみる！

Reply:
遅くなって申し訳ないけど、ここ `btn-circle` は丸いボタン用の css として用意しているからあてないでほしい
`btn-light` を上書きするんじゃなくって新しいクラス作って上書きしてください